python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -83,7 +83,7 @@ Notes:
 
 |     Model    |  FP32 Accuracy | QAT INT8 Accuracy | Accuracy Diff |
 |:------------:|:----------------------:|:----------------------:|:---------:|
-|   Ernie      |      79.76%            |         79.28%         |    -0.48% |               
+|   Ernie      |      79.76%            |        79.96%         |  +0.20%   |               
```

Comment:
it is `+0.25%`

Reply:
Why...
https://github.com/PaddlePaddle/benchmark/pull/319#discussion_r387363849